### DISCUSSION TITLE
How to increment the value of an attribute, based on the count of related rows in another table?

### DISCUSSION THREAD

**Discussion by DarkPhoenix2704** [prisma/prisma#16219] [Q&A] [answered]
How to increment the value of an attribute, based on the count of related rows in another table?

**Comment by nurul3101** [comment, marked as answer]
Hey @DarkPhoenix2704 👋 
Prisma supports [atomic operations](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#atomic-number-operations) like increment/decrement, but they would operate on the same attribute of the table. For incrementing based on count of related row in other table you would need to resort to [Raw Query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw).

Let me know if you face any issues in implementing raw query.

**Reply by DarkPhoenix2704** [reply]
Thank you for the answer.

**Comment by DarkPhoenix2704** [comment]


**Reply by nurul3101** [reply]
Hey @DarkPhoenix2704 👋 
Do you mind creating a new GitHub Discussion for this question?
It would be easier to search for users who have the same question, answering here would not get much visibility.

Thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
